Feat(enhancement): Tribute warning (#8076)
endless-sky/endless-sky: data/help.txt

help "friendly disabled"
	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`

help "tributing"

> 	`You are about to tribute a planet. When tributing a planet you will have to battle against a defense fleet and if you win you will have unrestricted access to this planet forever and will receive a small amount of money every day. Tributing is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want the canon experience of this game you should not tribute a planet.`

Reviewer: ```suggestion
	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat it's defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`
```
`Tributing` sounds like you're the one giving tribute. I'm pretty sure `tribute` is not a verb, `to tribute` doesn't sound right.

Reviewer 2: it's defense fleet -> its

> 	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat its defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`

Reviewer: ```suggestion
	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat it's defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`
```
As I said before in response to ThrawnCA, or:
```suggestion
	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat their defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`
```

> 	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat it's defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`

Author: ```suggestion
	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat its defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`
```

help "friendly disabled"
	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`

help "tributing"
	`You fire a warning shot, and the fourth wall shudders and cracks. An automated message appears on your computer.`

> 	`"Captain! Demanding tribute from planets is not integrated with the storyline! You will receive a daily income and gain unrestricted landing access to this location, at the cost of being considered an outlaw by their government, but any campaign missions will not take this into account, which may result in story events that don't make sense. Are you sure you want to proceed?"`

Reviewer 3: ```suggestion
	`"Captain! Demanding tribute from planets is not integrated with the storyline! You will receive a daily income and gain unrestricted landing access to this location, at the cost of being considered an outlaw by their government. Campaign missions will not take this into account, which may result in story events that don't make sense. Are you sure you want to proceed?"`
```
I think that sentence was way too long.

Reviewer 2: I think it does need to have a "but" or "however" or similar, though. The point is to say, "you will get these gameplay effects, _but_ you could also get these immersion-breaking effects."

It could still be split into two sentences, starting the second with "However,"

	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`

help "tributing"

> 	`You are about to tribute a planet. When tributing a planet you will have to battle against a defense fleet and if you win you will have unrestricted access to this planet forever and will recieve a small amount of money every day. Tributing is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want the cannon experience of this game you should not tribute a planet.`

Reviewer 2: "cannon" -> "canon"

Author: ```suggestion
	`You are about to tribute a planet. When tributing a planet you will have to battle against a defense fleet and if you win you will have unrestricted access to this planet forever and will recieve a small amount of money every day. Tributing is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want the canon experience of this game you should not tribute a planet.`
```

	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`

help "tributing"

> 	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat their defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`

Reviewer: Hahaha, you made the right choice. I misread it :)

Author: 👍
